translations/en-us.yaml

```diff
@@ -55,3 +55,14 @@ oss-components:
     error:
       title: Error
       subtitle: Failed to copy to your clipboard. Please try again.
+  post_types:
+    facebook_status: Facebook Status
+    tweet: Tweet
+    pin: Pin
+    instagram_media: Instagram Media
+    tiktok_video: Tiktok Video
+    youtube_video: Youtube Video
+    article: Article
+    story: Story
+    twitch_stream: Twitch Stream
+    any: Any
```

Comment:
you can remove this one as it has a meaning in Hyperstream but not really here :)

Reply:
Removed in 7efe4bdae28b942efbe5199b58241d1fe8a7b321